qpsolvers/solvers/qpax_.py

```diff
@@ -0,0 +1,68 @@
+from typing import Optional
+
+import numpy as np
+
+import qpax
+
+
+from ..problem import Problem
+from ..solution import Solution
+from ..conversions import linear_from_box_inequalities, split_dual_linear_box
+
+
+def qpax_solve_problem(
+    problem: Problem,
+    initvals: Optional[np.ndarray] = None,
+    verbose: bool = False,
+    backend: Optional[str] = None,
+    **kwargs,
+) -> Solution:
```

Comment:
We should put some documentation for this function. (It's very similar between solvers, don't hesitate to look at the others.)